dev-docs/source/MVPDesign.rst

```diff
@@ -179,6 +179,68 @@ in C++, or `unittest.mock
 can set expectations in the unit tests for certain methods to be
 called, and with certain arguments.
 
+MVP Template tool
+#################
+
+The `template.py script <https://github.com/mantidproject/mantid/blob/main/tools/MVP/template.py>`__
```

Comment:
Is probably obvious, but these instructions should be run under mantid-developer environment. is it worth adding this comment somewhere?